### PR TITLE
Purchase

### PR DESCRIPTION
**Pull request by danielopez26**
Buy something on the automationpractice website

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@
             java-version: '11'
             distribution: 'adopt'
         - name: Build with Maven
-          run: mvn clean test -Dbrowser="${{matrix.browser}}"
+          run:  mvn clean test -Dtest="TestGoogle"#purchase -Dbrowser="${{matrix.browser}}"
```
Comment (aperdomob) [inline .github/workflows/github-actions.yml:22]: why do you need pass this parameter

File: src/test/java/TestGoogle.java [hunk 2]
```diff
@@ -1,18 +1,25 @@
 import drivers.Drivers;
```
Comment (aperdomob) [inline src/test/java/TestGoogle.java:1]: the idea is keet the google test

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (danielopez26) [reply .github/workflows/github-actions.yml:22]: I have specified which test I want to run, I still have the previous exercise as test

File: src/test/java/TestGoogle.java (hunk 2 above)
Comment (danielopez26) [reply src/test/java/TestGoogle.java:1]: I'm not sure what you're telling me, I have the two tests and run in chrome and firefox

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (aperdomob) [reply .github/workflows/github-actions.yml:22]: la idea es que corran todos y que no tengas que estar agregandolos aquí

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: src/test/java/TestGoogle.java [hunk 3]
```diff
@@ -27,8 +30,56 @@ public void testSearch(){
         Assertions.assertEquals("YouTube", "YouTube");
     }
 
+    @Test
+    public void purchase() throws InterruptedException {
```
Comment (aperdomob) [inline src/test/java/TestGoogle.java:34]: Google y AutomationPractice no tienen relación por tanto cada uno debería estar en una clase (archivo) diferente

**Review by aperdomob — CHANGES_REQUESTED** [review]

**Review by aperdomob — APPROVED** [review]

**Pull request merged**